wmigroup/Tests/test_wmi_config.dsc.yaml

```diff
@@ -0,0 +1,13 @@
+# Example configuration for reading data from Windows WMI
+$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2023/10/config/document.json
+resources:
+- name: Get info from WMI
+  type: DSC/WMIGroup
+  properties:
+    resources:
+    - name: Get OS Info
+      type: root\cimv2/Win32_OperatingSystem
+    - name: Get OS Info
+      type: root\cimv2/Win32_BIOS
+    - name: Get OS Info
+      type: root\cimv2/Win32_Processor
```

Comment:
Technically, these type names violate our permitted pattern for [fully qualified type names][01]:

```regex
^\w+(\.\w+){0,2}\/\w+$
```
```Syntax
<owner>[.<group>][.<area>]/<name>
```

I wonder if this could be `root.cimv2/Win32_OperatingSystem` or `cimv2/Win32_OperatingSystem`.

[01]: https://learn.microsoft.com/powershell/dsc/reference/schemas/definitions/resourcetype?view=dsc-3.0&preserveView=true

Reply:
Replaced with dots.